src/app/components/bitcoin-custom-fee/hooks/use-bitcoin-custom-fee.tsx

```diff
@@ -15,11 +15,15 @@ import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/marke
 export const MAX_FEE_RATE_MULTIPLIER = 50;
 
 interface UseBitcoinCustomFeeArgs {
-  amount: number;
+  amountInSat: number;
```

Comment:
Can we use `Money`? It exists to solve this problem of not knowing which unit is being used

Reply:
hm, yeah, we can